mk_verificator/tests/system/mirgation/test_mirgation.py

```diff
@@ -0,0 +1,66 @@
+import novaclient.client as client
+import mk_verificator.utils as utils
+
+import time
+
+
+def wait_for_vm_status_is_active(client, vm_id, retry_delay=5, timeout=60):
+    timeout_reached = False
+    start_time = time.time()
+    while not timeout_reached:
+        vm_status = client.servers.find(id=vm_id).status
+
+        if vm_status == 'ACTIVE':
+            break
+        elif vm_status == 'ERROR':
+            assert False, "VM is in the status 'error'"
+
+        time.sleep(retry_delay)
+        timeout_reached = (time.time() - start_time) > timeout
+    else:
+        assert vm_status == 'ACTIVE', \
+               "VM didn\'t get to status active, " \
+               "VM is in the status {}".format(vm_status)
+
+
+def checking_vm_host(client, vm_id, target_host):
+    wait_for_vm_status_is_active(client, vm_id)
+    host = client.servers.find(id=vm_id)._info['OS-EXT-SRV-ATTR:host']
+
+    assert host == target_host, "VM didn't migrate"
+
+
+def test_migration():
+    config = utils.get_configuration(__file__)
```

Comment:
I would suggest to use special setup and teardown functions: 
Here you can find examples: http://pythontesting.net/framework/pytest/pytest-fixtures-easy-example/